python has some gcc-apple specific CFLAGS built in... rip them out

--- configure.in.orig	2007-03-05 21:35:42 +0100
+++ configure.in	2007-03-05 21:35:58 +0100
@@ -796,7 +796,6 @@
 	    ;;
 	# is there any other compiler on Darwin besides gcc?
 	Darwin*)
-	    BASECFLAGS="$BASECFLAGS -Wno-long-double -no-cpp-precomp -mno-fused-madd"
 	    if test "${enable_universalsdk}"; then
 		BASECFLAGS="-arch ppc -arch i386 -isysroot ${UNIVERSALSDK} ${BASECFLAGS}"
 	    fi
